src/device-state/current-state.ts

```diff
@@ -172,12 +171,41 @@ const report = _.throttle(async () => {
 	}
 }, constants.maxReportFrequency);
 
+/**
+ * Listen for config changes to disableHardwareMetrics
+ * to avoid a db query every 10 seconds
+ */
+let disableHardwareMetrics: boolean;
+(async () => {
+	await config.initialized;
+	config.on('change', (conf) => {
```

Comment:
I don't like this, we are already querying the database every 10 seconds for other configurations such as local mode, api timeout, etc. I think we should step away from using the config module this way as it makes code more complex and harder to read.